core/src/main/java/org/elasticsearch/index/query/ScriptQueryParser.java

```diff
@@ -65,7 +61,8 @@ public ScriptQueryBuilder fromXContent(QueryParseContext parseContext) throws IO
             } else if (token == XContentParser.Token.START_OBJECT) {
                 if (parseContext.parseFieldMatcher().match(currentFieldName, ScriptField.SCRIPT)) {
                     script = Script.parse(parser, parseContext.parseFieldMatcher());
-                } else if (parseContext.parseFieldMatcher().match(currentFieldName, PARAMS_FIELD)) { // TODO remove in 3.0 (here to support old script APIs)
+                } else if (parseContext.parseFieldMatcher().match(currentFieldName, PARAMS_FIELD)) {
+                    // TODO remove in 3.0 (here to support old script APIs)
```

Comment:
this TODO is not good :( but it was there already. We should probably remove this stuff in another PR.